### ISSUE TITLE
BNavItemDropdown position props are not working

### ISSUE THREAD

**Issue by alexdohm** [bootstrap-vue/bootstrap-vue-next#659]
**Describe the bug**
The props dropup, dropleft, dropright, right, left props are not working for BNavItemDropdown

**Expected behavior**
Proper positioning of dropdown with applied props. Even though they are applied in the code, they do not work

```
     <b-nav-item-dropdown class="user-dropdown" right>
        <template #button-content> User </template>
        <b-dropdown-item>Settings</b-dropdown-item>
        <b-dropdown-item>Sign Out</b-dropdown-item>
      </ b-nav-item-dropdown>
```

<img width="869" alt="Screenshot 2022-09-13 at 14 20 54" src="https://user-images.githubusercontent.com/26983611/189899475-0b645ae2-b0c6-471a-a922-d4a41c6c473e.png">

**Environment (please complete the following information):**
"bootstrap": "^5.2.0",
"bootstrap-vue-3": "^0.3.3",

**Comment by VividLemon** [comment]
From my testing, it appears that the menu inside the navbar is not able to go right, because it doesn't have enough space... I've noticed that it modifies the space of the navbar component. When I had built the component, it was all rendered in a simple b-nav, so no problems. But navbar spacing is static and doesn't like to move, so issues. This is all incorrect as the menu is rendered as a _child_ of navbar, but actual affect is appeared to be _outside_ the component. Complicated issue
